applications/matter_bridge/doc/matter_bridge_description.rst

```diff
@@ -137,21 +142,52 @@ For details, see the `Testing`_ section.
 User interface
```

Comment:
In the line 137 the exact button is mentioned, which if I understand correctly will be affected by the same issue with DK numbering. Can this part be generalized?
For example, ``If the Bluetooth LE advertising times out, you can re-enable it manually (see the :ref:`matter_bridge_app_ui` section). ``